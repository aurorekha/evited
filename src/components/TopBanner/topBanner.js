import React from 'react';
import './topBanner.scss';

function TopBanner() {
  return(
    <div className="topBanner">
      <h1>You are (E)Vited!</h1>
    </div>
  )
}

export default TopBanner;
