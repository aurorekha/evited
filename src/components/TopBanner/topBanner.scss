@font-face {
  font-family: "aulyars";
  src: url("../../fonts/aulyars/aulyars_italic-webfont.woff") format("woff"),
}

.topBanner {
  font-family: 'aulyars', Arial, sans-serif;
  color: white;
  height: 50vh;
  width: 100%;
  font-size: 4rem;
  background-size: cover;
  background-image: url('../../images/topBannerOne.jpg');
  background-attachment: fixed;
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  padding-top: 15%;
  span {
    font-size: 1rem;
  }
}

h1 {
  margin-top: 0;
}
