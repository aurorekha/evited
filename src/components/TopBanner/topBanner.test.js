import React from 'react';
import TopBanner from './topBanner';
import renderer from 'react-test-renderer';

it('renders TopBanner section correctly', () => {
  const tree = renderer.create(
    <TopBanner />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
