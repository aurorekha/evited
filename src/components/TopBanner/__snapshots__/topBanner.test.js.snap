// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TopBanner section correctly 1`] = `
<div
  className="topBanner"
>
  <h1>
    You are (E)Vited!
  </h1>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="topBanner"
>
  <h1>
    You are (E)Vited!
  </h1>
</div>
`;
